test/runtests.jl

```diff
@@ -118,6 +118,16 @@ ch2 = chain(1:0, 1:2:5, 0.2:0.1:1.6)
 @test eltype(ch2) == typejoin(Int, Float64)
 @test collect(ch2) == [1:2:5; 0.2:0.1:1.6]
 
+ch3 = chain(1:10,1:10, 1:10)
+@test length(ch3) == 30
+@test Base.iteratorsize(ch3) == Base.HasLength()
```

Comment:
`iteratorsize` is imported (if available) and otherwise defined in this package, so it shouldn't be prefixed with `Base`. (Causing tests to fail.)